src/index.js

```diff
@@ -60,7 +60,7 @@ const microdown = function () {
       ),
 
       // lists
-      /(?:(^|\n)([+-]|\d+\.) +(.*(\n[ \t]+.*)*))+/g,
+      /(?:(^|\n)(?!(?:\* )+\*)([*+-]|\d+\.) +(.*(\n[ \t]+.*)*))+/g,
```

Comment:
I didn't include * in the list style to prevent collisions with inline styling. 
I guess your first ?! condition is to prevent the hr from being listified.
We could ether
use ` *** => <hr>    and    * *test* => <li><strong>test</strong</li>`
or remove remove the * for the list.